### PR TITLE
Hughsie/f41

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@superm1 what magic am I missing please?

**Comment by superm1** [comment]
> @superm1 what magic am I missing please?

I haven't moved the Windows build job to Github Actions because of how fragile it's been and how we rely upon it for publishing EXEs and documentation.  Those parts need to come too if it moves...

**Comment by hughsie** [comment]
>  Those parts need to come too if it moves

I *think* I just did that

File: .github/workflows/matrix.yml [hunk 1]
```diff
@@ -115,3 +115,37 @@ jobs:
...
+        path: |
+          ${{ github.workspace }}/dist/setup/*.msi
+          ${{ github.workspace }}/dist/VERSION
+          ${{ github.workspace }}/dist/news.txt
+
+  publish-windows:
+    runs-on: ubuntu-latest
+    needs: build-windows
+    container:
+      image: circleci/golang:1.17.5
```
Comment (superm1) [inline .github/workflows/matrix.yml:140]: I doubt that Github actions is going to have the circlei image..
We probably need to use a Github actions style of publishing releases

**Review by superm1 — APPROVED** [review]
That seems to work (but we can't check I got the publishing 100$ right until we tag).

**Comment by superm1** [comment]
I am going to rebase my other PR on this to maybe axe circle all together.  So I'm going to merge this ones.

**Comment by hughsie** [comment]
Thanks Mario, many thanks!

**Pull request merged**